tests/test_resolver.py

```diff
@@ -329,35 +328,23 @@ def test_compile_failure_shows_provenance(resolver, from_line):
             "test_package==1.2",
             "test_package==1.2.0",
             True,
-            marks=pytest.mark.skipif(
-                PIP_VERSION[:2] < (20, 2), reason="Required only for pip>=20.2"
-            ),
         ),
         pytest.param(
             "test_package>=1.2",
             "test_package>=1.2.0",
             True,
-            marks=pytest.mark.skipif(
-                PIP_VERSION[:2] < (20, 2), reason="Required only for pip>=20.2"
-            ),
         ),
         ("test_package[foo,bar]==1.2", "test_package[bar,foo]==1.2", True),
         ("test_package[foo,bar]>=1.2", "test_package[bar,foo]>=1.2", True),
         pytest.param(
```

Comment:
Now that `marks` isn't required, can drop `pytest.param` and use a tuple (like the other lines). This pattern appears multiple times and could be cleaned up throughout the diff.

Reply:
Thanks for noticing! Addressed in d9ab55df38221ef8c0f0af237d095c0ce3e65db1.

Reply:
:+1: LGTM